src/sdl2_compat.c

```diff
@@ -3239,15 +3239,25 @@ SDL_GetRenderDriverInfo(int index, SDL_RendererInfo *info)
 DECLSPEC SDL_Renderer *SDLCALL
 SDL_CreateRenderer(SDL_Window *window, int index, Uint32 flags)
 {
+    SDL_Renderer *renderer = NULL;
     const char *name = NULL;
+    int want_tartgettexture = flags & SDL2_RENDERER_TARGETTEXTURE;
```

Comment:
`want_targettexture`?

Reply:
Took me some time to spot the typo, but it is fixed now.